deploy/webhook.yaml

```diff
@@ -0,0 +1,111 @@
+---
```

Comment:
I would move this file into `./webhook` dir since it's related to webhook deployment